src/context/UserContext.tsx

```diff
@@ -0,0 +1,159 @@
+import {
+  createContext,
+  useContext,
+  useReducer,
+  ReactNode,
+  Dispatch,
+} from 'react';
+
+enum Currency {
```

Comment:
Create a separate file for this enum and move it to `types/Currency.ts`

Reply:
I moved Currency enum to types folder and import to UserContect file.